button-examples/Android/android-button-example.md

```diff
@@ -95,8 +102,8 @@ A text button has a text label, a transparent container and an optional icon.
 ![Text button anatomy diagram](assets/text_button_anatomy.png)
 
 1. Text label
-1. Container
```

Comment:
FYI, GitHub's MD renderer should automatically renumber ordered lists:

1.
1.
1.
Converts to
1.
2.
3.